services/ui-src/src/components/export/StickyBanner.tsx

```diff
@@ -0,0 +1,43 @@
+import { Image, Box, Button } from "@chakra-ui/react";
+import pdfIcon from "assets/icons/icon_pdf_white.png";
+
+export const StickyBanner = () => {
```

Comment:
Let's pass in some text props to this so it doesn't have to just be for MCPAR (planning for MLR/NAAAR needing this too).